docs/DEVELOPER_GUIDE.md

```diff
@@ -333,5 +333,12 @@ Ideally, the `version` specified should match the release tag of the plugin.
 This helps users and maintainers to easily identify which version of the plugin
 they have installed.
 
+### Automating the releasing of new version
+
+You can use `github actions` to automate the release process. e.g. You can use
```

Comment:
Don’t put github actions in code font?

Reply:
No need for e.g. here. You aren’t giving an example.

Reply:
done